### PR TITLE
docs: expand `adapter-node` polyfill docs

### PR DESCRIPTION
**Pull request by coryvirok**
Add some extra info to the adapter-node polyfill option's docs.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

File: documentation/docs/25-build-and-deploy/40-adapter-node.md [hunk 1]
```diff
@@ -157,7 +157,9 @@ node build
 
 ### polyfill
 
-Controls whether your build will load polyfills for missing modules. It defaults to `true`, and should only be disabled when using Node 18.11 or greater.
+Controls whether your build will load polyfills for missing modules. It defaults to `true`, and should only be disabled when using Node 18.11 or greater.
```
Comment (Conduitry) [inline documentation/docs/25-build-and-deploy/40-adapter-node.md:160]:
```suggestion
Controls whether your build will load polyfills for missing modules. It defaults to `true`, and should only be disabled when using Node 18.11 or greater.
```

**Comment by Conduitry** [comment]
How ready for prime time is nodejs.dev? I haven't been following that. I do notice that their homepage currently says that the most recent LTS is 18.15.0 (not 18.16.0) and that the latest is 19.8.1 (not 20.0.0), which doesn't inspire tremendous confidence. I'd prefer linking to the docs on nodejs.org for now.

File: documentation/docs/25-build-and-deploy/40-adapter-node.md [hunk 2]
```diff
@@ -157,7 +157,9 @@ node build
 
 ### polyfill
 
-Controls whether your build will load polyfills for missing modules. It defaults to `true`, and should only be disabled when using Node 18.11 or greater.
+Controls whether your build will load polyfills for missing modules. It defaults to `true`, and should only be disabled when using Node 18.11 or greater. 
+
+Note: even with Node 18.11 or greater, you may still need to use experimental flags. E.g. `--experimental-global-webcrypto` to enable the `crypto` global. More info on which globals require flags can be found [here](https://nodejs.dev/en/api/v18/globals/).
```
Comment (benmccann) [inline documentation/docs/25-build-and-deploy/40-adapter-node.md:162]: It makes it sound like others may require a flag, but I don't think they do. I think we could write it a bit more precisely as:
```suggestion
Note: to use Node's built-in `crypto` global with Node 18 you will need to use the `--experimental-global-webcrypto` flag. This flag is not required with Node 20.
```

**Comment by coryvirok** [comment]
> How ready for prime time is nodejs.dev? I haven't been following that. I do notice that their homepage currently says that the most recent LTS is 18.15.0 (not 18.16.0) and that the latest is 19.8.1 (not 20.0.0), which doesn't inspire tremendous confidence. I'd prefer linking to the docs on nodejs.org for now.

Honestly, I'm not sure. It was just the first place Google took me to when I was investigating the issue and it looked reputable. 🙃

**Review by benmccann — APPROVED** [review]

**Pull request merged**